### PR TITLE
[BEAM-14049][Playground] Add new API method to get Precompiled Object

### PR DESCRIPTION
**Pull request by AydarZaynutdinov**
[BEAM-14049]
Add and implement new API method to get Precompiled Object by cloud path

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by AydarZaynutdinov** [comment]
R: @KhaninArtur @ilya-kozyrev

**Review by pavel-avilov — APPROVED** [review]
Overall, LGTM. Just small suggestions.

File: playground/api/v1/api.proto [hunk 1]
```diff
@@ -274,6 +284,9 @@ service PlaygroundService {
   // Get all precompiled objects from the cloud storage.
   rpc GetPrecompiledObjects(GetPrecompiledObjectsRequest) returns (GetPrecompiledObjectsResponse);
 
+  // Get precompiled objects from the cloud storage.
```
Comment (pavel-avilov) [inline playground/api/v1/api.proto:287]:
```suggestion
  // Get precompiled object from the cloud storage.
```

Comment (AydarZaynutdinov) [reply playground/api/v1/api.proto]: Done

**Comment by AydarZaynutdinov** [comment]
retest this please

**Review by KhaninArtur — APPROVED** [review]
LGTM

**Comment by AydarZaynutdinov** [comment]
R: @pabloem

**Comment by pabloem** [comment]
lgtm thanks. Can you please add more detail in the bug (BEAM-14049) as to why this is necessary? (I suppose the backend needs to get the examples for the catalog. this will be useful in the future for maintenance

**Pull request merged**